### PR TITLE
refactor: create utils module

### PR DESCRIPTION
**Pull request by IyadhKhalfallah**
---

- [ ] This PR references an open issue
- [ ] Added a test
- [ ] Updated the documentation
- [ ] Added a screenshot of a successful customer deployment

### CODE REVIEW

File: now/utils/implementation/helpers.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import signal
```
Comment (joschkabraun) [inline now/utils/implementation/helpers.py:1]: what does `implementation` refer to?

File: now/utils/flow/helpers.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import yaml
```
Comment (joschkabraun) [inline now/utils/flow/helpers.py:1]: Let's call it `jcloud` instead of `flow` as these utils are needed for `jcloud`

Comment (IyadhKhalfallah) [reply now/utils/flow/helpers.py:1]: Sure!

File: now/utils/common/helpers.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
+import signal
+import sys
+from collections.abc import MutableMapping
+from inspect import stack
+from typing import Any
+
+from pyfiglet import Figlet
+
+
+def debug(msg: Any):
```
Comment (violenil) [inline now/utils/common/helpers.py:10]: Is this function even used anywhere?

File: now/utils/common/helpers.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+class BetterEnum:
+    def __iter__(self):
+        return [getattr(self, x) for x in dir(self) if ('__' not in x)].__iter__()
+
+
+sigmap = {signal.SIGINT: my_handler, signal.SIGTERM: my_handler}
+
+
+def print_headline():
```
Comment (violenil) [inline now/utils/common/helpers.py:62]: Can this function not be moved to the dialog.py? It is not used anywhere else in the codebase.

**Review by violenil — CHANGES_REQUESTED** [review]
Looks good! Nice refactoring.

**Review by joschkabraun — APPROVED** [review]
LGTM, nice refactoring!

**Review by violenil — APPROVED** [review]
🚀

**Pull request merged**